### DISCUSSION TITLE
Bekanntgabe / Weitergabe Repo

### DISCUSSION THREAD

**Discussion by jsuesse** [importPI19fromDHGE/dhge-pi19-sem4#82] [Ideas]
1. **Wollen wir das Repo unter PI20, PI21 bekannt machen?** Gibt es Bedenken wegen dem veröffentlichten Material, vielleicht stößt das manchen Dozenten sauer auf - kann uns eigentlich egal sein, allerdings sind gute Beziehungen zu Dozenten nie schlecht!
2. **Falls ja: Wie gehen wir vor?**

Ich stelle mir Folgendes vor,  diese Instanz als "Original"-Repo bleibt erhalten, mehr oder weniger inaktiv / archiviert. Keine neuen Schreibzugriffe, PR für nicht-PI19.

Sponate Idee ohne viel drüber Nachdenken: Vielleicht das alte Repo privat nehmen und einigen PI'lern Zugang ermöglichen: Spuren verwischen. Kommen Probleme mit, oder? Erstmal egal.

Neue Jahrgänge kopieren / forken das Repo unter neuem Namen mit neuer Organisation, können die Aufzeichungen so erweitern. Wenn sich Dozenten ändern gibt's neue Ordner. 
Man müsste auch den Hinweis mitgeben, dass gelegentlich aufgeräumt werden soll, damit nicht zu viele Artefakte entstehen.

Nach 2-5 Jahren werden die Repos richtig archiviert, weil outdated. 

Steht und fällt halt alles damit, dass die Jahrgänge immer die engagierten Leute des jüngeren Jahrgangs finden und diese Verwaltung weiter so aktiv betrieben wird.

**Comment by Steve2955** [comment]
Das Repo kann gerne für sämtliche andere Matrikel bekannt gemacht werden. So bin ich in der Vergangenheit auch immer verfahren. Das Projekt wurde meinerseits nie *"geheim gehalten"*. Zu jüngeren Matrikeln unseres Studiengangs hatte ich aber bisher keinen Kontakt. 

Ich bin der Meinung, dass es auf jeden Fall offen und für alle frei zugänglich bleiben sollte. Bedenken bezüglich den Dozenten habe ich eigentlich keine. Es ist doch vollkommen normal, dass solche Informationen auch Matrikel übergreifend geteilt werden (ist nicht ganz so öffentlich, aber meiner Meinung nach genau das Gleiche nur weniger fair). 

Wichtig ist es halt wirklich ein paar engagierte Studierende in den jüngeren Matrikeln zu finden. Ob wir zu einer allgemeinen DHGE-Orga migrieren und die Repos weiter betreut werden, oder andere Matrikel ihre eigene Orga mit Forks bekommen ist mir ziemlich egal. Archivieren ist bei letzterer Option aber eine gute Idee.

**Reply by jsuesse** [reply]
Ich habe allein durch die Firma die Möglichkeit viele PI-ler anzusprechen. Irgendjemand motiviertes finden wir dann schon

**Reply by Steve2955** [reply]
Also aus meiner Sicht, kannst du gerne andere Matrikel ansprechen. @RvNovae @ZeroPointMax @sthinbetween Was haltet ihr davon?

**Reply by RvNovae** [reply]
Ich fänds auch gut, wenn die nachfolgenden Matrikel noch was von dem Ganzen hier haben. 
Habe allerdings nicht wirklich Kontakt zu den PI20 Studenten in meiner Firma.

**Reply by emkZero** [reply]
Gehe ich auf jeden Fall mit; wie die genaue "Implementierung" dann aussieht, da bin ich offen. @Steve2955 deine Philosophie gefällt mir sehr gut, den Gedanken mit den Forks von @nukerxy würde ich gerne mitnehmen.

Früher oder später werden Profs das hier sehen (es braucht ja nur jmd auf GH nach "dhge" suchen) - bin gespannt, wie die Meinungen zu den Repos stehen. (Wie dem auch sei, sie müssen sich mmn damit abfinden, dass Studis sowas machen, va im IT-Bereich. Wenn wir es nicht machen, dann früher oder später jemand anders :D)

**Reply by jsuesse** [reply]
Gut, die Main-Contributor sind dafür :D -  Ich schreibe jetzt die ersten an.

**Comment by LordofAgents** [comment]
Moinsen, PI20er hier. Vielen Dank für das Weiterleiten! Ich würde das an den gesamten PI20-Discord weiterleiten, wenn das euch recht ist?
Ich muss allerdings dazu sagen, dass ich nicht denke, dass viele das weiterpflegen. Viele nutzen OneNote, und haben sich dort Strukturen geschaffen, die über die DHGE-Services mit anderen Studenten geteilt werden. Ich sehe eher die Gefahr, dass Studenten, die nicht so ganz schreibtüchtig sind, das GitHub als Faulheitsplan-B nutzen.

**Reply by jsuesse** [reply]
genau, mach das mal so. 

Muss ich mir das mit OneNote so vorstellen, dass ihr quasi Notizbücher habt, in Teams bzw. auf dem DHGE-Sharepoint und die habt ihr dann freigegeben für andere.? 
Kann man auch machen. Ich weiß nicht, wie gut OneNote dahingehend geeignet ist und kann es daher nicht vergleichen. Nach Eingwöhnung mit Git klappt hier fast alles sehr gut. Gits Versionverwaltung ist sehr nett, alle Änderungen sind nachverfolgbar, können einzeln revertet werden, müssen von Kommilitonen überprüft werden. Alles ist (aktuell) öffentlich, wer Lust hat kann teilnehmen.
Ob die Studierenden diese Option nutzen, bisschen Zeit in Git investieren, ist ihre Sache. 

Der Lernerfolg ist meiner Erfahrung nach ziemlich gering, wenn man die Sachen nur liest und zuvor nicht mitgearbeitet und aufgepasst hat. Es kann den ein oder anderen retten, aber wer keine Ambitionen hat, dem wird es auch nicht viel bringen. Ich persönlich denke in den Übungs-Fächern lieber aktiv mit, als ständig mitzuschreiben bzw. wenn ich mitschreibe, dann auf Papier. Andere machen es gleich digital und fügen es dem Repo hinzu. Danach kann man hier gemeinsam eine Nachbereitung machen.

**Reply by LordofAgents** [reply]
Zum OneNote: Ja, die meisten sind Einzelkämpfer, es bilden sich halt kleine Grüppchen, die sich über den Sharepoint das freigeben. Ich gebs mal weiter und antworte, wenn ich Feedback habe.

Zum Erfolg: Ja, da hast du schon Recht, ich persönlich höre meist zu und verlasse mich auf Script und kleine Ergänzungen, nur in Digitaltechnik und Elektrotechnik schreibe ich voll mit mangels Skript und lesbare Notizen vom Dozenten.

**Reply by emkZero** [reply]
OneNote ist gut, ja (solange nicht mehrere auf einer Seiter rumtreten :D). Ich persönlich habe dann im Nachgang erst für GitHub aufbereitet. Per Hand schreiben hilft ja echt, fand ich dahingehend angenehmer (außer Programmierfächer, das kotzt ja an xD).

Mit den Mitmachern ist ja noch nichts in Stein gemeißelt. Angefangen hat damit glaube ich @Steve2955, danach ist das so vor sich hin eskaliert :-)
So finden sich bei euch evtl. auch noch ein paar, die (evtl ja auch zusätzlich zum OneNote) mitmachen wollen.

Bzgl Leecher: sollen sie ja sehen, wie weit sie damit kommen, hat ja @nukerxy schön gesagt.

Insgeheim ist das ein Geschäftsmodell: wir klatschen einfach überall GPL3 oder CC dran und wenn jemand ohne Credit / Mitmachen das Zeug nutzt, verklagen wir den bis auf die Knochen und werden reich oder so 😂

**Comment by jsuesse** [comment]
#116 beendet das Thema
